doc/tntcxx_api.rst

```diff
@@ -28,141 +26,184 @@ Methods:
 
 .. _tntcxx_api_connector_connect:
 
-.. c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
+..  c:function:: int connect(Connection<BUFFER, NetProvider>& conn, const std::string_view& addr, unsigned port)
 
-   Connects to a Tarantool instance that is listening on ``addr:port``.
+    Connects to a Tarantool instance. If a host doesn't reply within a timeout
```

Comment:
What was wrong with "that is listening on `addr:port`"?

Also, what is the host? Is it the same as instance?

Reply:
Nothing wrong. The idea was to have a short description for the method and move all the details in the description of params. But now I did as you proposed in another comment: method's logic is explained here and in params and return sections we have shorter description.